src/main/converter/opds.ts

```diff
@@ -150,9 +153,9 @@ export class OpdsFeedViewConverter {
         };
     }
 
-    public convertOpdsFeedToView(feed: OPDSFeed): OpdsResultView {
+    public async convertOpdsFeedToView(feed: OPDSFeed): Promise<any> {
```

Comment:
Would it be possible to avoid using `any` here? Same for the calling side (`public async browse(data: any): Promise<any> {}` in `src/main/api/opds.ts`)